modules/core/src/views/first-person-view.js

```diff
@@ -40,3 +41,7 @@ export default class FirstPersonView extends View {
 }
 
 FirstPersonView.displayName = 'FirstPersonView';
+
+FirstPersonView.defaultController = {
```

Comment:
Q: Is it better to have this static, rather than as a `View.getController()`?

Static methods have side effects. I think tree shakers struggle with them. We might even want to reconsider whether the `displayName`s here are needed. I think we don't use them, we could cut.